### ISSUE TITLE
Fix version conflict in Python dependencies of analyst image

### ISSUE THREAD

**Issue by trey0** [nasa/isaac#136]
Here's the relevant error:
https://github.com/nasa/isaac/actions/runs/7835253902/job/21380266683

We can unblock getting PRs to pass CI again once #135 is merged (temp disables the analyst build as part of the PR testing). This ticket is for a permanent fix that allows the image to be built again.

Note: Solving this issue is not as urgent as some others because the analyst notebook is not needed for ISS ops!

**Comment by trey0** [comment]
- [ ] Remember to re-enable analyst image build in CI before merging this fix
